src/Nethermind/Nethermind.Runner/Nethermind.Runner.csproj

```diff
@@ -3,23 +3,20 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
     <LangVersion>latest</LangVersion>
-    <ServerGarbageCollection>true</ServerGarbageCollection>
-    <ConcurrentGarbageCollection>true</ConcurrentGarbageCollection>
-    <TargetLatestRuntimePatch>true</TargetLatestRuntimePatch>
```

Comment:
Path doesn't matter as is self contained (or self built) and new version always published?

Reply:
Yep, see [here](https://learn.microsoft.com/en-us/dotnet/core/deploying/runtime-patch-selection).